src/custom-element.js

```diff
@@ -1112,7 +1112,7 @@ function createBaseCustomElementClass(win) {
       const win = this.ownerDocument.defaultView;
       const event = win.document.createEvent('Event');
       event.data = data;
-      event.initEvent(name, true, true);
+      event.initEvent(name, /* bubbles */ true, /* cancelable */ true);
```

Comment:
Slightly larger task since `CustomEvent`'s uses `detail`, not `data`. Could be part of the GFI's scope.